src/transaction/txParser.ts

```diff
@@ -183,15 +238,20 @@ const parseTxCerts = (txCertificates: any[]): _Certificate[] => {
   )
 }
 
-const parseTxWithdrawals = (
-  withdrawals: any,
-): _Withdrawal[] => {
+const parseTxWithdrawals = (withdrawals: any): _Withdrawal[] => {
   if (!isWithdrawalsMap(withdrawals)) {
     throw Error(Errors.WithrawalsParseError)
   }
   return Array.from(withdrawals).map(([address, coins]): _Withdrawal => ({ address, coins: BigInt(coins) }))
 }
 
+const parseValidityIntervalStart = (validityIntervalStart: any): BigInt => {
+  if (isBigNumber(validityIntervalStart)) {
+    return BigInt(validityIntervalStart)
+  }
+  throw Error(Errors.ValidityIntervalStartParseError)
```

Comment:
In all the other functions, we do 
```
if(!isSomething) {
   throw Error
}
return something
```
Here we do it in the opposite order, just to stay consistent.

Reply:
Refactored